numpyro/distributions/copula.py

```diff
@@ -0,0 +1,74 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+from jax import lax, numpy as jnp
+
+import numpyro.distributions.constraints as constraints
+from numpyro.distributions.continuous import MultivariateNormal, Normal
+from numpyro.distributions.distribution import Distribution
+from numpyro.distributions.util import is_prng_key, validate_sample
+
+
+class GaussianCopula(Distribution):
+    arg_constraints = {
+        "correlation_matrix": constraints.corr_matrix,
+        "correlation_cholesky": constraints.lower_cholesky,
+    }
+    reparametrized_params = [
+        "correlation_matrix",
+        "correlation_cholesky",
+    ]
+
+    def __init__(
+        self,
+        marginal_dist,
+        correlation_matrix=None,
+        correlation_cholesky=None,
+        validate_args=None,
+    ):
+        self.marginal_dist = marginal_dist
+        self.base_dist = MultivariateNormal(
+            covariance_matrix=correlation_matrix,
+            scale_tril=correlation_cholesky,
+        )
+        self.normal = Normal()
```

Comment:
I guess this is not needed. We can use `Normal().cdf/icdf` in the `sample` and `log_prob` method.